sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -363,8 +390,9 @@ def _collect_plans(self, rows, cursor, deadline):
                         1,
                         **self.check.debug_stats_kwargs(tags=["error:obfuscate-xml-plan-{}".format(type(e))])
                     )
-
-                tags = self.check.tags + ["db:{}".format(row['database_name'])]
+                tags = list(self.check.tags)
+                if 'database_name' in row:
+                    tags += ["db:{}".format(row['database_name'])]
```

Comment:
this doesn't have to do with this PR, but why do we add this `db` tag here instead of just grabbing from the payload in the backend and adding the tag to the metrics there?

Reply:
Good question; we can update it subsequently